definition/rest/v1/omnichannel.ts

```diff
@@ -8,6 +9,8 @@ import { ISetting } from '../../ISetting';
 import { PaginatedRequest } from '../helpers/PaginatedRequest';
 import { PaginatedResult } from '../helpers/PaginatedResult';
 
+type booleanString = 'true' | 'false';
```

Comment:
I didn't get y we couldn't use boolean type here instead of this?

Reply:
Well, I'd like to use a boolean, but the usage of this is for string values of true/false (as endpoints are using strings).

We should modify that tho, but, I didn't feel that was part of this PR